### PR TITLE
Added scripts to create different types of pools and different storage classes corresponding to pool type

### PR DESCRIPTION
**Pull request by sushma1118**
* Bash scripts are added to create different types of pools and storage classes.
* Using sed operation, the litmus job label, litmus job name, pool type, pool name and storage class names are changed in litmus/providers/cstor/cstor-block-device-pool/run_litmus_test.yml and it is applied to create different types of pools.

Some files are added, some are modified:
	modified:   .gitlab-ci.yml
	new file:   Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool
	new file:   Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-pool
	new file:   Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/G45J-create-mirrored-pool/create-mirrored-pool
	deleted:    Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/HC5P-create-striped-pool/create-pool
	new file:   Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/HC5P-create-striped-pool/create-striped-pool

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -29,12 +29,33 @@ K9YC-OpenEBS:
...
+  stage: OPENEBS-SETUP
+  script:
+    - chmod 775 ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool
+    - ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool
+
+HC5P-RAIDZ2-POOL:
+  image: openebs/openshift-ci:latest
+  stage: OPENEBS-SETUP
+  script:
+    - chmod 775 ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-poola
```
Comment (nsathyaseelan) [inline .gitlab-ci.yml]: @sushma1118 there is typo in name `pool`

File: Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+   ---------------------------------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/app: cstor-block-device-pool-provision/app: cstor-raidz1-pool-provision/g' \
+-e 's/generateName: cstor-block-device-pool-provision/generateName: cstor-block-device-raidz1-pool-provision/g' \
+-e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' \
+-e 's/value: cstor-block-disk-pool/value: cstor-block-disk-pool-raidz1' \
+-e 's/value: striped/value: raidz' \
+-e 's/value: openebs-cstor-disk/value: openebs-cstor-raidz1' create_raidz1_pool.yml
+
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool]: @sushma1118 where is the sed command to append the run_id.

File: .gitlab-ci.yml (hunk 1 above)
Comment (sushma1118) [reply .gitlab-ci.yml]: OK

File: Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool (hunk 2 above)
Comment (sushma1118) [reply Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool]: OK. Done

**Pull request merged**